src/Security/SslContext.cs

```diff
@@ -523,7 +523,7 @@ public SslStatus SetEncryptionCertificate (SecIdentity identify, IEnumerable<Sec
 			get {
 				IntPtr value;
 				result = SSLCopyPeerTrust (Handle, out value);
-				return (value == IntPtr.Zero) ? null : new SecTrust (value);
+				return (value == IntPtr.Zero) ? null : new SecTrust (value, true);
```

Comment:
I'm not sure if I'm understanding your answer. My question is around a situation like this:

```
var x = new SecTrust(handle, true);

// do some stuff

x.Dispose();

```
May be a misuse of it, but it also may not be obvious that handle is going to get an extra CFRelease inside Dispose.